src/library.js

```diff
@@ -2736,6 +2736,7 @@ mergeInto(LibraryManager.library, {
   emscripten_pc_get_function: (pc) => {
 #if !USE_OFFSET_CONVERTER
     abort('Cannot use emscripten_pc_get_function without -sUSE_OFFSET_CONVERTER');
+    return 0;
```

Comment:
Without this I was setting a closure warning/error when attempting to convert the return value of this function to a BigInt.